pkg/resources/view_acceptance_test.go

```diff
@@ -16,31 +24,42 @@ import (
 	"github.com/hashicorp/terraform-plugin-testing/tfversion"
 )
 
-func TestAcc_View(t *testing.T) {
-	viewId := acc.TestClient().Ids.RandomSchemaObjectIdentifier()
-	accName := viewId.Name()
-	query := "SELECT ROLE_NAME, ROLE_OWNER FROM INFORMATION_SCHEMA.APPLICABLE_ROLES"
-	otherQuery := "SELECT ROLE_NAME, ROLE_OWNER FROM INFORMATION_SCHEMA.APPLICABLE_ROLES where ROLE_OWNER like 'foo%%'"
+func TestAcc_View_basic(t *testing.T) {
+	_ = testenvs.GetOrSkipTest(t, testenvs.EnableAcceptance)
+	acc.TestAccPreCheck(t)
```

Comment:
this should be in the resource.Test (check other tests)

Reply:
Yes, but when I run one test instead of the whole suite, the test default objects are not created before all the policies, and this test fails.